lib/hsa/mcwamp_hsa.cpp

```diff
@@ -2215,13 +2268,9 @@ class HSAQueue final : public KalmarQueue
             }
         }
 
-
-        // GC for finished kernels
-        if (asyncOps.size() > ASYNCOPS_VECTOR_GC_SIZE) {
-            DBOUTL(DB_RESOURCE, "asyncOps size=" << asyncOps.size() << " exceeds collection size, compacting");
-            asyncOps.erase(std::remove(asyncOps.begin(), asyncOps.end(), nullptr),
-                         asyncOps.end());
-        }
+        /* We used to garbage collect nullptr entries from asyncOps here.
```

Comment:
nit: use `//` in all comments, both single-line and multi-line.

Reply:
Done.